frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.scss

```diff
@@ -10,6 +10,10 @@
   margin-top: 0;
 }
 
+.lso-create-lvs__max-disk-limit {
+  max-width: 100px;
+}
+
```

Comment:
Let's remove this as well and use the other class.

Reply:
Okay... using single class for min disk size, max disk size and max disk limit...